public/main.js

```diff
@@ -124,6 +131,7 @@ function reloadTodoList() {
         } else {
             clearCompleteButton.style.visibility = "hidden";
         }
+        todoListRefreshTimerId = setTimeout(reloadTodoList, todoListRefreshRate);
```

Comment:
That was my original plan; the advantage of doing it this way is that the refresh timer is reset by the call to "clearTimeout" at the top of reloadTodoList, and the next timer is only set when the todo list finishes refreshing. This means that the client won't try to get the list again right after (or during) a refresh of the list, such as when the list is changed by some user action or a particularly slow previous timed refresh.